.github/ISSUE_TEMPLATE.md

```diff
@@ -10,8 +10,11 @@ Feature suggestions should be described [in the forum Features category](https:/
 
 -->
 
-#### Software and hardware versions 
-Collect v1.x.x, Android vx.x.x, device used...
+### ODK Collect version
```

Comment:
Did you make this text bigger than Problem description/Expected behavior/Other information on purpose or is it a mistake?

Reply:
That's a mistake. Will fix.